packages/passport-auth/lib/Passport.js

```diff
@@ -0,0 +1,304 @@
+const passport = require('passport');
+
+const FIELD_SERVICE_NAME = 'service';
+const FIELD_USER_ID = 'serviceUserId';
+const FIELD_USERNAME = 'serviceUsername';
+const FIELD_TOKEN_SECRET = 'tokenSecret';
+const FIELD_ITEM = 'item';
+
+let isInitialized = false;
+
+class PassportAuthStrategy {
+  constructor(authType, keystone, listKey, config, ServiceStrategy = null) {
+    this.authType = authType;
+    this.keystone = keystone;
+    this.listKey = listKey;
+    this.config = {
+      tokenSecretField: FIELD_TOKEN_SECRET,
+      itemField: FIELD_ITEM,
+      sessionListKey: `passportSession`,
+      useSession: false,
+      sessionIdField: 'passport',
+      keystoneSessionIdField: 'keystone_passport',
+      scope: [],
+      ...config,
+    };
+    this.ServiceStrategy = ServiceStrategy;
+
+    this.createSessionList();
+
+    this.passportStrategy = this.getPassportStrategy();
+    passport.use(this.passportStrategy);
```

Comment:
We can remove this line now.

Reply:
We would have to call it sometime, Isn't it? Where should I add this then?

Reply:
we can not remove this line `passport.use(this.passportStrategy);` passportjs would fail. I have to add it somewhere. In my tests it has no impact on whether it is called before `passport.initialize` or after

Reply:
this does not affect `express` middleware, I think it should be fine here.